editor/debugger/editor_debugger_inspector.cpp

```diff
@@ -280,7 +280,29 @@ String EditorDebuggerInspector::get_stack_variable(const String &p_var) {
 	for (KeyValue<StringName, Variant> &E : variables->prop_values) {
 		String v = E.key.operator String();
 		if (v.get_slice("/", 1) == p_var) {
-			return variables->get_variant(v);
+			Variant var = variables->get_variant(v);
+
+			String prefix = "";
+			String suffix = "";
+
+			switch (var.get_type()) {
+				case Variant::STRING:
+					prefix = "\"";
+					suffix = "\"";
```

Comment:
You can use `String::quote()` so you wouldn't need the `suffix`, and just need to handle the `&` and `^` prefixes.

Reply:
Done